spec/javascript/apiService.test.js

```diff
@@ -22,7 +22,7 @@ describe('apiService', () => {
       var expectedData = {'lease': lease}
 
       var result = apiService.createOrUpdateLease(lease, fakeAppId)
-      expect(result).resolves.toEqual(true)
+      expect(result).resolves.toEqual(undefined)
```

Comment:
All of these promises resolve to `undefined` as the tests sit right now whether we resolve them ahead of the expect or not

Reply:
Hm so that means that these tests might all just be not working 😦. I'd rather get them to work and return the right value then set the expectation to undefined.